### PR TITLE
Stage to Test

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 1]
```diff
@@ -82,6 +82,7 @@
                             <div class="inner-container">
                                 <div
                                     class="show-on-small-screen"
+                                    [ngClass]="{ 'pending-sales-balance-due': isPendingSales }"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:85]: please check this class is added

**Review by prateek-giddh — COMMENTED** [review]
Done

File: apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html [hunk 2]
```diff
@@ -1523,6 +1521,7 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                                 class="form-control"
                                                 [ngModel]="vm.selectedLedger.voucherNo"
                                                 [disabled]="true"
+                                                [style.width.px]="80"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html:1524]: please remove this, we have removed this from test, other PR are not merged yet, therefore this line is there on stage

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 3]
```diff
@@ -467,7 +467,7 @@
                                         [placement]="'top'"
                                         (click)="getBankTransactions()"
                                     >
-                                        <i class="icon-refresh lh-0 font-18"></i>
+                                        <i class="icon-refresh lh-0"></i>
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/ledger/ledger.component.html:470]: please add this class font-18, it is removed

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html (hunk 1 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:85]: @aditi-tiwari17 yes, Shalinee has added this class in SCSS for responsive layout, but the class was not mentioned in HTML

File: apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html (hunk 2 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html]: Done

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**